### ISSUE TITLE
FR: Document downloads with images

### ISSUE THREAD

**Issue by gerroon** [nortikin/sverchok#2305]
Hi

It would be nice if the download version of the documentation comes with images embedded. It seems like I am not seeing any images because all of them are loading from unreachable urls. My desktop is not connected, so that highlighted the issue. 

It is not a biggie I guess I can download the whole site, but it would be nice if the download offers the full package.

thanks

**Comment by zeffii** [comment]
https://readthedocs.org/projects/sverchok/downloads/pdf/latest/

not sure how up to date that is

**Comment by zeffii** [comment]
yeah, i also don't see images on that..    sorry, don't know @gerroon

**Comment by gerroon** [comment]
@zeffii 

Thanks. The pdf option is nice but like you said that is missing the images as well.

**Comment by zeffii** [comment]
@gerroon  any of these ? https://readthedocs.org/projects/sverchok/downloads/

**Comment by gerroon** [comment]
sure I can download the Pdf for instance, but there is no images in it. I think all the images are referenced from dropbox etc so the builder is not including them

**Comment by gerroon** [comment]
The epub works but it is because it is still loading images from external sites. See the link from an epub page

https://cloud.githubusercontent.com/assets/619340/3375992/3bbc0c86-fbd0-11e3-9456-353c77fd0d17.gif

**Comment by gerroon** [comment]
I just cloned his whole site and images are still broken. I think httrack did not fix the image urls properly. I downloaded the images as well, but they do not inline. Ohh well

**Comment by zeffii** [comment]
what about https://github.com/nortikin/sverchok/wiki  (offline docs there as a Tar ?)

**Comment by gerroon** [comment]
It does not look like it comes with the images which is the main issue

**Comment by vicdoval** [comment]
Hi @gerroon i have just copied the whole docs site using WinHTTrack, it worked great. I compressed the copy and uploaded it to my google drive [here](https://drive.google.com/open?id=1BmmDfDydxxT0XqPWRD-nCvWA1mzFBUlm)
It is not a pdf but with it you can navigate offline through the docs.
Maybe @nortikin could host it somewhere at http://nikitron.cc.ua ...

**Comment by nortikin** [comment]
ok, thanks @vicdoval

**Comment by nortikin** [comment]
http://nikitron.cc.ua/sverchok_ru.html
http://nikitron.cc.ua/sverchok_en.html
done

**Comment by vicdoval** [comment]
@nortikin I used  [Httrack](https://www.httrack.com/page/1/en/index.html) to create the local copy 😃
